style.css

```diff
@@ -0,0 +1,3 @@
+h1 {
+    color: red;
+}
```

Comment:
- [ ] Also, there are issues with the linters checks. I would suggest fixing these in order to meet the project requirements.
I think the problem with Webhint is that you have to add the HTML boilerplate :wink:
![image](https://user-images.githubusercontent.com/81522435/143053816-ff6a20f1-efda-4700-ac8a-e9c52609ff18.png)
And the problem with Stylelint is because in line 2 there should be 2 spaces for indentation (I think you are using 4) and you are missing an empty line at the end of the file :smiley:
![image](https://user-images.githubusercontent.com/81522435/143054126-f64e8986-ce64-43b8-8dcd-df6ce406e769.png)

Reply:
I think I need some help with this
I don't know how to solve like this problems yet

Reply:
may be I solved these problems.
because I have at the bottom of this page in green:  
All checks have passed
3 successful checks
@github-actions
Linters / Lighthouse (pull_request) Successful in 1m Details
@github-actions Linters / Webhint (pull_request) Successful in 47s Details
@github-actions Linters / Stylelint (pull_request) Successful in 22s

Reply:
Yes! You already solved them :muscle:

Reply:
What is my next step?
Should I submit again in dashboard?

Reply:
Yes, if you have worked on the suggested changes you should request a review again :smiley: